gordo_components/data_provider/azure_utils.py

```diff
@@ -30,8 +29,11 @@ def get_datalake_token(
     lib.DataLakeCredential
         A lib.DataLakeCredential which can be used to authenticate towards the datalake
     """
-    logger.info("Looking for ways to authenticate with data lake")
-    if dl_service_auth_str:
+
+    if interactive:
+        logger.info("Attempting to use interactive azure authentication")
+        return lib.auth()
```

Comment:
I think adding a simple test with patching here an asserting this is called would be sufficient. https://pypi.org/project/pytest-mock/

Reply:
Fixed